slides/setup.R

```diff
@@ -34,6 +35,8 @@ knitr::opts_chunk$set(
 
 # devtools::install_github("gadenbuie/countdown")
 library(countdown)
+library(emo)
```

Comment:
The Quarto group lost their minds when they realized that we were just pasting emojis into the notes. We should make them programmatically; otherwise it can mess up some of the quarto rendering.